infra/zookeeper-deployment.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Since Zookeeper keeps data for coordination, it must be presented as a `statefulset`.